lib/import-export-visitor.js

```diff
@@ -114,9 +125,14 @@ module.exports = class ImportExportVisitor extends Visitor {
     const directives = parent.directives;
     if (directives) {
       const directiveCount = directives.length;
-
       for (let i = 0; i < directiveCount; ++i) {
-        insertCharIndex = Math.max(directives[i].end, insertCharIndex);
+        const d = directives[i];
```

Comment:
Yep, Acorn represents directives as string-valued `ExpressionStatement`s, but that difference only matters from the AST perspective. The generated code looks exactly the same. In other words, if `this.modifyAST` can be assumed to be false, there should be no difference in behavior.